src/renderer/components/+catalog/catalog.tsx

```diff
@@ -186,6 +187,26 @@ export class Catalog extends React.Component<Props> {
     );
   };
 
+  renderName(item: CatalogEntityItem<CatalogEntity>) {
+    const disabledIcon = HotbarStore.getInstance().entityPinnedToHotbar(item.entity);
+
+    return (
+      <div className={styles.entityName}>
+        {item.name}
+        <Icon
+          small
+          disabled={disabledIcon}
+          className={styles.pinIcon}
+          material="push_pin"
+          tooltip={disabledIcon ? "" : "Pin to Hotbar"}
+          onClick={(event) => {
+            event.stopPropagation();
+            this.addToHotbar(item);
+          }}/>
```

Comment:
_nit_: we could use also `src/renderer/utils/prevDefault.ts` to prevent event propagation:

```suggestion
          onClick={prevDefault(() => this.addToHotbar(item))}
```

Reply:
Also I would better to allow these icons to **unpin** icon instead of no-action when pinned already:

<img width="806" alt="Screenshot 2021-09-22 at 23 09 14" src="https://user-images.githubusercontent.com/6377066/134414752-b762d940-5c46-43fd-958d-443398dd74da.png">